setup.py

```diff
@@ -1,97 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-import os
-import re
-import sys
-
-from setuptools import setup
-
-
-def get_version(package):
-    """
-    Return package version as listed in `__version__` in `init.py`.
-    """
-    init_py = open(os.path.join(package, '__init__.py')).read()
-    return re.search("__version__ = ['\"]([^'\"]+)['\"]", init_py).group(1)
-
-
-def get_packages(package):
-    """
-    Return root package and all sub-packages.
-    """
-    return [dirpath
-            for dirpath, dirnames, filenames in os.walk(package)
-            if os.path.exists(os.path.join(dirpath, '__init__.py'))]
-
-
-def get_package_data(package):
-    """
-    Return all files under the root package, that are not in a
-    package themselves.
-    """
-    walk = [(dirpath.replace(package + os.sep, '', 1), filenames)
-            for dirpath, dirnames, filenames in os.walk(package)
-            if not os.path.exists(os.path.join(dirpath, '__init__.py'))]
-
-    filepaths = []
-    for base, filenames in walk:
-        filepaths.extend([os.path.join(base, filename)
-                          for filename in filenames])
-    return {package: filepaths}
-
-
-def get_long_description(long_description_file):
-    """
-    Read long description from file.
-    """
-    with open(long_description_file, encoding='utf-8') as f:
-        long_description = f.read()
-
-    return long_description
-
-
-
-version = get_version('apistar')
-
-
-setup(
-    name='apistar',
-    version=version,
-    url='http://www.encode.io/apistar/',
-    license='BSD',
-    description='Blazingly fast & beautifully expressive Web APIs',
-    long_description=get_long_description('README.md'),
-    long_description_content_type='text/markdown',
-    author='Tom Christie',
-    author_email='tom@tomchristie.com',
-    packages=get_packages('apistar'),
-    package_data=get_package_data('apistar'),
-    install_requires=[
-        'click',
-        'jinja2',
-        'requests',
-        'pyyaml',
-        'werkzeug',
-        'whitenoise'
-    ],
-    classifiers=[
```

Comment:
where did all the classifiers go?

Reply:
actually in PKGINFO for tar.gz and METADATA for the wheel.
For now it only provide : 
```
Classifier: Programming Language :: Python :: 3
Classifier: Programming Language :: Python :: 3.5
Classifier: Programming Language :: Python :: 3.6
Classifier: Programming Language :: Python :: 3.7
```
I don't know if it's possible to add others. I'll look at it if the this PR stay opened.